lib/user.rb

```diff
@@ -0,0 +1,33 @@
+require './lib/joke'
+
+class User
+attr_reader :name
+attr_accessor :jokes
+
+  def initialize(name)
+    @name = name
+    @jokes = []
+  end
+
+  def learn(joke)
+    # joke = Joke.new(id, setup, punchline)
+    joke = joke
+     # @jokes.each do |joke|
+
+    # @jokes << joke
```

Comment:
this commented line on 17 is correct! all you need to do is take the joke and add it to the @jokes array.